Description: Upstream changes introduced in version 1.0.0+git5314+gf0eab15-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 qtmobility (1.0.0+git5314+gf0eab15-1) experimental; urgency=low
 .
   * Initial release (Closes: #nnnn)
 .
 The person named in the Author field signed this changelog entry.
Author: Fathi Boudra <fabo@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- qtmobility-1.0.0+git5314+gf0eab15.orig/src/multimedia/multimedia.pro
+++ qtmobility-1.0.0+git5314+gf0eab15/src/multimedia/multimedia.pro
@@ -107,10 +107,6 @@ SOURCES += qmediacontrol.cpp \
     qvideorenderercontrol.cpp \
     qmediatimerange.cpp
 
-include(audio/audio.pri)
-include(video/video.pri)
-include(effects/effects.pri)
-
 mac {
    HEADERS += qpaintervideosurface_mac_p.h
    OBJECTIVE_SOURCES += qpaintervideosurface_mac.mm
